moveit_ros/moveit_servo/src/utils/common.cpp

```diff
@@ -255,6 +255,22 @@ void updateSlidingWindow(KinematicState& next_joint_state, std::deque<KinematicS
   joint_cmd_rolling_window.push_back(next_joint_state);
 }
 
+/**
+ * @brief Composes a Float64MultiArray message for controllers requiring Float64MultiArray input.
+ * 
+ * This function converts the given joint state into a Float64MultiArray message.
```

Comment:
these two sentences virtually say the same thing -- you probably can combine these into just a single line.